server/tests/test.py

```diff
@@ -68,8 +68,8 @@ def test_retrieve_file_key_not_found():
 
 
 def test_update_files_success():
-    key = "test_key"
-    new_key = "new_test_key"
+    key = "test_key1"
```

Comment:
assigned different key due to conflicts with upload
as now upload will not override the existing old files